internal/cli/actions.go

```diff
@@ -138,7 +139,7 @@ auth0 actions show <id>`,
 
 			if err := ansi.Waiting(func() error {
 				var err error
-				action, err = cli.api.Action.Read(url.PathEscape(inputs.ID))
```

Comment:
`url.PathEscape` is not necessary for action IDs, except when building the URL for `auth0 actions open`.